### PR TITLE
CPLP-1313-DbAudit App

### PR DESCRIPTION
**Pull request by qxz2mqe**
https://jira.catena-x.net/browse/CPLP-1313

### CODE REVIEW

File: src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs [hunk 1]
```diff
@@ -1,24 +1,4 @@
...
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- *
- * SPDX-License-Identifier: Apache-2.0
- ********************************************************************************/
-
-// <auto-generated />
+﻿// <auto-generated />
```
Comment (ntruchsess) [inline src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs:1]: please manually (re-)add license-header

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
+﻿using System;
```
Comment (ntruchsess) [inline src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.cs:1]: please manually add license-header

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.Designer.cs [hunk 3]
```diff
@@ -0,0 +1,5576 @@
+﻿// <auto-generated />
+using System;
```
Comment (ntruchsess) [inline src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.Designer.cs:2]: please manually add license-header

**Review by ntruchsess — CHANGES_REQUESTED** [review]

File: src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs (hunk 1 above)
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.Migrations/Migrations/PortalDbContextModelSnapshot.cs]: fixed

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.cs (hunk 2 above)
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.cs]: fixed

File: src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.Designer.cs (hunk 3 above)
Comment (qxz2mqe) [reply src/portalbackend/PortalBackend.Migrations/Migrations/20221010092958_CPLP-1313-AuditOffer.Designer.cs:22]: fixed

**Comment by ntruchsess** [comment]
somehow the merge of main caused all changes from main to re-appear in the PR resulting in a whole bunch of conflicts. Looks as if the merge was done as a squash-commit.
I therefore I did restart on a new branch cherry-picking the existing commits recreating the migration: https://github.com/catenax-ng/product-portal-backend/pull/267
I therefore close this PR as redundant

**Pull request closed without merging**